app.js

```diff
@@ -0,0 +1,10 @@
+const { getTopics } = require("./controllers/controller.js");
+const express = require("express");
+const app = express();
+const db = require("./db/connection");
+
+app.use(express.json());
```

Comment:
We do not need app.use(express.json()) yet, so please remove it for now & re-add it once we need it :)